src/content/components/modal/usage.mdx

```diff
@@ -3,32 +3,43 @@ title: Modal
 tabs: ['Code', 'Usage', 'Style']
 ---
 
+### Modals communicate information via a secondary window and allow the user to maintain the context of a particular task.
+
 <AnchorLinks>
 
 - [General guidance](#general-guidance)
 - [Format](#format) 
 - [Variations](#variations) 
-- [Dismissal](#dismissal)
 
 </AnchorLinks>
 
 ## General guidance
 
-_Modals_ communicate information via a secondary window and allow the user to maintain the context of a particular task. Use modals sparingly because they interrupt user workflow.
+Modals interrupt user workflow by design. They are most effective when a task must be completed before a user can continue. While effective when used correctly, modals should be used sparingly to limit disruption to a user experience.
+
+#### Dismissal
+
+Modals may be dismissed in 3 ways:
+
+- Using the “✕” in the upper right-hand corner of the modal
+- Pressing the `ESC` key
+- Clicking or touching outside of the modal
```

Comment:
I'd like to double check with @carbon-design-system/design on this. It makes sense that Passive modal can be dismissed by clicking outside the modal, Transactional modal should probably not be dismissed so easily, in case of accidental clicks outside before task completion. If everyone agrees, I can make an issue for dev fix.

Reply:
We've talked about making changes, including an acknowledgment for passive modals. We should make an issue and open it up for discussion.

Reply:
Yeah modals in general need a pass. But thats a good point, transactional probably shouldn't have the "click anywhere to close" function